components/browser/menu/src/test/java/mozilla/components/browser/menu/item/BrowserMenuHighlightableItemTest.kt

```diff
@@ -36,7 +36,7 @@ class BrowserMenuHighlightableItemTest {
             android.R.drawable.ic_menu_report_image,
             android.R.color.black,
             android.R.color.black,
-            BrowserMenuHighlightableItem.Highlight(android.R.drawable.ic_menu_report_image, android.R.color.holo_red_light)
+            BrowserMenuHighlightableItem.Highlight(android.R.drawable.ic_menu_report_image, android.R.color.holo_red_light, android.R.color.holo_red_light)
```

Comment:
I'm surprised we're using holo themed colours. Don't we want to use our own ui-colors from photon?

Reply:
I made a mistake, this is for a test so shouldn't matter.